hedera-node/hedera-addressbook-service-impl/src/main/java/com/hedera/node/app/service/addressbook/impl/schemas/V053AddressBookSchema.java

```diff
@@ -101,7 +101,6 @@ public void migrate(@NonNull final MigrationContext ctx) {
                     .accountId(nodeInfo.accountId())
                     // Default node description hard coded to the values used currently
                     .description(formatNodeName(nodeInfo.nodeId()))
-                    .gossipEndpoint(nodeInfo.gossipEndpoints())
```

Comment:
Other option would be to add `internal` and `external` gossipEndpoints into `NodeInfo`. Since we discussed `gossipEndpoints ` are not used anywhere other than `AddressBookSchema` I am okay removing them, since by the time we enable DAB, we could iterate through `Roster` instead of `AddressBook`

Reply:
But Genesis startup will run this migrate 🤔 and in future PRs we also need to sue this in TSS while constructing new candidate Rosters and we will not have enough information with this removed. Is it possible to add both `internal` and `external` gossip endpoints to NodeInfo @anthony-swirldslabs ?

Reply:
You can not remove gossipEndpoint here. gossipEndpoint is node's important attribute and is used. When DAB enabled, we will still need another migration to get this attribute.